docs/acp/user/developing-an-app/setting-up-message-broker.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Setting Up the Message Broker
+Descriptions: Learn how to develop an app
+template: howto-guide-template
+redirect_from:
+- /docs/acp/user/developing-an-app.html
+---
+
+To enable the message broker functionality, you must install a special branch of the `spryker/message-broker` by running the following command:
+```bash
+docker/sdk cli composer require "spryker/message-broker:dev-beta/localstack-replacement as 1.8.0"
+```
+
+## Update the Code
+
+For testing the application, update or add the `MessageBrokerDependencyProvider` as shown in the code snippet below.
+
+```php
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
```

Comment:
Please remove this doc block from the example.

Reply:
++